content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js/angular/core/main.js

```diff
@@ -0,0 +1,30 @@
+/*global angular*/
+
+/*
+ * This is the definition of the application module itself as well as core configurations like routing and the http service.
+ * Since this file defines that module that all of our controllers and services are part of, it needs to be listed first in the clientlibs js.txt file.
+ */
+var sampleModule = angular.module('angularjs-sample', ['ngRoute']);
+
+/*
+ * The routeProvider configuration maps URL paths to AJAX content to be loaded into the ng-view container defined in the containerPage component.
+ * Note that the templateUrls are each using a .content Sling selector.  This allows us to switch out the content portion of the containerPage with that of the partials.
+ */
```

Comment:
Yup.  I have updated the comment to be more clear.  I'm just validating
everything before I push again.

On Sun, Mar 8, 2015 at 11:41 AM, david g. <notifications@github.com> wrote:

> In
> content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js/angular/core/main.js
> <https://github.com/Adobe-Consulting-Services/acs-aem-samples/pull/26#discussion_r26007636>
> :
>
> > @@ -0,0 +1,30 @@
> > +/*global angular*/
> > +
> > +/*
> > + * This is the definition of the application module itself as well as core configurations like routing and the http service.
> > + * Since this file defines that module that all of our controllers and services are part of, it needs to be listed first in the clientlibs js.txt file.
> > + */
> > +var sampleModule = angular.module('angularjs-sample', ['ngRoute']);
> > +
> > +/*
> > + * The routeProvider configuration maps URL paths to AJAX content to be loaded into the ng-view container defined in the containerPage component.
> > + * Note that the templateUrls are each using a .content Sling selector.  This allows us to switch out the content portion of the containerPage with that of the partials.
> > + */
>
> Yup - i get that; just want it to be clear to less familiar folks reading
> through :) .. If i was new to this i would think "what is this magic
> .content selector being used, why is it being used, can i use a different
> one, how do i implement it, etc." .. the simple answer is its a selector
> that binds to a content.jsp script on the page that is responsible for
> rendering only the markup necessary for that view and not any HTML
> header/footer data.
>
> Thats easy for you and I to know that, but if youve never implemented a
> single page angular app in AEM its not totally straightforward.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Adobe-Consulting-Services/acs-aem-samples/pull/26/files#r26007636>
> .
>